libs/ui/src/colors.ts

```diff
@@ -0,0 +1,163 @@
+import { transparentize } from 'polished'
+import { css } from 'styled-components/macro'
+
+/**
+ * Central color system for CoW Protocol applications
+ *
+ * Colors are organized into several categories:
+ * 1. Neutral colors (neutral0-100): Base grayscale palette
+ * 2. CoW Protocol colors (cowfi_): Brand-specific colors
+ * 3. CoW AMM colors (cowamm_): AMM-specific colors
+ * 4. Explorer colors (explorer_): Explorer-specific colors
+ *
+ * Usage:
+ * import { Color } from '@cowprotocol/ui'
+ * color: ${Color.neutral100}
+ */
+export const Color = {
+  // Neutral colors - Base grayscale palette from black (0) to white (100)
+  neutral100: '#FFFFFF',
+  neutral98: '#FFF8F7',
+  neutral95: '#FFEDEC',
+  neutral90: '#F0DEDE',
+  neutral80: '#D4C3C2',
+  neutral70: '#B8A7A7',
+  neutral60: '#9C8D8D',
+  neutral50: '#827474',
+  neutral40: '#685B5B',
+  neutral30: '#504444',
+  neutral20: '#382E2E',
+  neutral10: '#23191A',
+  neutral0: '#000000',
+
+  // TODO(theme-cleanup): These colors below were migrated
+  // TODO(theme-cleanup): They should be reviewed and potentially consolidated with the existing color system.
```

Comment:
I see your TODO and yes, this is the key part of the refactoring.
Anyway, one thing we should fix now it's naming, we don't use snake_case.
Please rename them to camelCase or even make the object hierarchical: 
```
const Color = {
    cowfi: {...},
    explorer: {...}
}
```

Also, I think it's implied by your TODO, but I want to clarify.
`libs/ui` is supposed to be a generic library which might be used outside of our projects. So, we should not operate by words like cowfi/explorer, the variables names must give a purpose of the value instead of the project where it's used.

Reply:
Agree, thank you!